src/pages/episode/sections/show-notes/person-notes.js

```diff
@@ -23,15 +23,13 @@ function PersonNotes({person}) {
       {
         nothing ?
           <p>No links, tips, or picks this week</p> :
-        (
```

Comment:
Then we might need to look into the eslint rule since that style will throw (on the line starting `<div...`):

`[eslint] Expected indentation of 6 space characters but found 8. (react/jsx-indent)`